pages/liquidity/[poolid].tsx

```diff
@@ -1,19 +1,26 @@
-import { observer } from "mobx-react";
-import { NextPage } from "next";
-import { useRouter } from "next/router";
-import React, { useEffect, useState } from "react";
-import { ChevronLeft, Info } from "react-feather";
+import { useQuery } from "@tanstack/react-query";
+import {
+  isAvailable,
+  isIndexedData,
+  NA,
```

Comment:
What's the purpose of `NA`?

Reply:
NA is for values that arent available in some particular context like pool volume if indexer isnt connected.